f2/src/summary/DevicesSummary.js

```diff
@@ -51,7 +51,7 @@ export const DevicesSummary = props => {
         {hasInfoToDisplay ? (
           <Stack as="dl" spacing={4}>
             <DescriptionListItem
-              descriptionTitle="confirmationPage.devices.deviceOrAccount"
+              descriptionTitle="confirmationPage.devices.device"
               description={devices.deviceOrAccount}
```

Comment:
Still not coming through - change this to `description={devices.device}`

Reply:
I think I've made the necessary change now :)